src/guides/v2.3/coding-standards/docblock-standard-general.md

```diff
@@ -575,17 +575,20 @@ interface MutableInterface
  */
 class LimitedMutableClass implements MutableInterface
 {
+    /**
+    * Returns integer value
+    */
     public function getVal(): int
     {
+        ...
     }
 
     /**
-     * Sets 0 in case a non-integer value is passed
-     *
-     * @param int|string|bool|float|null $value
+     * @inheritdoc
```

Comment:
Contradicts with
> The @inheritdoc tag SHOULD NOT be used.

Reply:
@lenaorobei Thanks for the feedback, I have removed `@inheritdoc`. please review